setup_environment.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+###################################################################
+# The purpose of this script is to download, install
+# and configure all files and programs required to
+# take part in developing ActivitySync (Android only).
+###################################################################
+
+CURRENT_USER=$USER
+
+# update package lists from repositories
+sudo apt-get update
+sudo apt-get install git
+
+sudo mkdir /usr/lib/jvm
+cd /tmp/
+
+# download, install and configure java7 jdk
+sudo wget --no-check-certificate --no-cookies --header "Cookie: oraclelicense=accept-securebackup-cookie" http://download.oracle.com/otn-pub/java/jdk/7u79-b15/jdk-7u79-linux-x64.tar.gz
```

Comment:
> What if this script will be repeated for a second time in some time (~6 months) from now ?
Will oracle remove JDK 7u79 at this point ?

JDK 7u79 is the last jdk7 version provided by Oracle. I don't know whether it will be removed or not (but I'm pretty sure it will last at least a few years on their servers).

> Does this script require constant maintenance or not ?

Yes, it does. Script should be checked (and updated if necessary) at least quarterly.

> Does Oracle provide -latest version always pointing to newest JDK ?

I couldn't find any way of updating jdk to latest version but using [ppa:webupd8team/java](http://askubuntu.com/questions/183867/how-do-i-update-oracle-java-7-jdk-and-jre/183868) repository. As you mentioned earlier on trello, we shouldn't use software from unknown sources.